astropy/units/function/core.py

```diff
@@ -227,7 +227,7 @@ def to(self, other, value=1., equivalencies=[]):
 
         Raises
         ------
-        UnitsError
+        :class:`~astropy.units.UnitsError`
```

Comment:
Here again I think the `:class:` is obvious from context.

Reply:
I agree, but apparently it links faster (doesn't need to run inference) and since space isn't an issue...

Reply:
Don't forget these docstrings are also shown when you do `u.Unit.to?` in ipython - so I think we should not add things just because they make it easier for sphinx/computers! For human users, I think the `:class:` only adds confusion, since of course what is really done is raise with an *instance* of `UnitsError`.

Reply:
✅